### DISCUSSION TITLE
vweb Livereload

### DISCUSSION THREAD

**Discussion by jacksonmowry** [vlang/v#18863] [Questions and Answers]
Hi there everyone!
I am trying to use the livereload feature of vweb for faster development but I am running into a bit of an issue. 

According to the documentation:
_Run your vweb app with a live reload via_ `v -d vweb_livereload watch run .`

_Now modifying any file in your web app (whether it's a .v file with the backend logic or a compiled .html template file) will result in an instant refresh of your app in the browser. No need to quit the app, rebuild it, and refresh the page in the browser!_

The issue is that any action that causes a db update (specifically for sqlite) will cause the server to reload. In my case I am updating a page view count every time the page is loaded which causes an infinite loop of page reloads. 

So my question is, is there any way to ignore a specific file through the livereload? or should I just move the db out of the folder that livereload is being run in?

**Comment by medvednikov** [comment]
I don't think live reload looks at the sqlite db file?

Try moving it to another folder, if that fixes your problem,  then it's an easy to fix bug in vweb.

**Comment by jacksonmowry** [comment]
After moving the .db into its own subfolder the live reload feature seems to ignore it. So live reload is looking for the .db if it is in the same folder as the code being run.

**Comment by spytheman** [comment]
> is there any way to ignore a specific file through the livereload?

Try `v -d vweb_livereload watch --ignore=.db run .`

See also `v watch -h` for more options, that the watch supports.
